tracing/providers/jaeger/src/main/java/io/helidon/tracing/providers/jaeger/JaegerTracerProvider.java

```diff
@@ -53,4 +61,8 @@ public JaegerTracerBuilder createBuilder() {
     public boolean available() {
         return true;
     }
+
+    static List<SpanListener> lifeCycleListeners() {
```

Comment:
Nit: `spanListeners`

Reply:
Yes. In fact, this method is no longer needed so I've removed it.